tests/integ/modin/resample/test_resample.py

```diff
@@ -19,7 +18,24 @@
 )
 from tests.integ.utils.sql_counter import SqlCounter, sql_count_checker
 
-agg_func = pytest.mark.parametrize("agg_func", IMPLEMENTED_AGG_METHODS)
+agg_func = pytest.mark.parametrize(
+    "agg_func",
+    [
+        "max",
+        "min",
+        "mean",
+        "median",
+        "sum",
+        "std",
+        "var",
+        "count",
+        "size",
+        "first",
+        "last",
+        "quantile",
+        "nunique",
+    ],
```

Comment:
for my education, what is the reason we duplicate the content of IMPLEMENTED_AGG_METHODS  here?  but the `IMPLEMENTED_AGG_METHODS` still defined `src/snowflake/snowpark/modin/plugin/_internal/resample_utils.py` and has the same content, right?

Reply:
We add "indices" to `IMPLEMENTED_AGG_METHODS` b/c in `snowflake_query_compiler.py::resample` we have a check like follows:

```
elif resample_method in IMPLEMENTED_AGG_METHODS:
      <perform resample>
else:
            ErrorMessage.not_implemented(
                f"Resample Method {resample_method} has not been implemented."
            )
```

Reply:
But you bring up a good point, it's a bit clunky to relist all of these, so instead did this in the test:

```
# Parametrize on all IMPLEMENTED_AGG_METHODS except 'indices' which is tested in a separate file
agg_func = pytest.mark.parametrize("agg_func", IMPLEMENTED_AGG_METHODS[:13])
```

Reply:
Thanks @sfc-gh-nkrishna for the update. I did later realize I missed the line you added "indices" to IMPLEMENTED_AGG_METHODS.  I think `IMPLEMENTED_AGG_METHODS[:13]` might be prone to make the test failed if others update the IMPLEMENTED_AGG_METHODS ,  we need to skip some agg methods for the rest, maybe consider what I suggested in my updated comment? 

```
agg_func = pytest.mark.parametrize(
    "agg_func",
     list(filter(lambda x: x not in  ["indices"], IMPLEMENTED_AGG_METHODS))
 )
```